docs/quickstart.md

```diff
@@ -108,13 +109,14 @@ npm install --save firebase reactfire
    //...
    ```
 
-## 6. Modify `src/App.js`
+## 6. Add the Burrito() function to `App.js`
+> Open the src directory and add code to App.js as described below.
 
-1. Import the `useFirestoreDocData` and `useFirestore` hooks
+1. Import the required ReactFire hooks
```

Comment:
`SuspenseWithPerf` isn't a hook, so we need to reword this. Maybe "Import from ReactFire" or "Import the following modules from ReactFire"?